packages/babel-parser/src/util/scope.js

```diff
@@ -170,13 +170,16 @@ export default class ScopeHandler<IScope: Scope = Scope> {
   }
 
   checkLocalExport(id: N.Identifier) {
+    const currentScope = this.currentScope();
+    const scope =
+      currentScope.flags & SCOPE_PROGRAM ? currentScope : this.scopeStack[0];
```

Comment:
Can `currentScope.flags & SCOPE_PROGRAM` ever be false?

Reply:
I seem to have made a mistake when I added this... 15c8306